readme.md

```diff
@@ -29,11 +29,19 @@ The plugin streamlines the entire configuration process and removes the need for
 - Jenkins 2.190.1+
 - Bitbucket Server 7.4+
 
-Note: Bitbucket Server 5.6 to 7.3 are also supported, but they're not recommended. This is because some plugin features are not available when using these versions. Instead, we recommend using Bitbucket Server 7.4+ as when you also set up an Application Link you will have access to all plugin features. 
+Note: Bitbucket Server 5.6 to 7.3 are also supported, but they're not recommended. This is because some plugin features are not available when using these versions. Instead, we recommend using Bitbucket Server 7.4+ so that you have access to all plugin features and you can set up an Application Link.
 
```

Comment:
@ali3-atlassian since the amount of info is growing, I think it's worth having a table of contents up front. Because of this I think we can also remove the numbers from the titles. 

```suggestion
## In this read.me

1. [Install the plugin](#install-the-plugin)
2. [Configure the plugin](#configure-the-plugin)
3. [Use the plugin](#use-the-plugin)
4. [Contribute to the plugin](#contribute-to-the-plugin)
```